features/itunes/components/TrackCard/tests/index.test.tsx

```diff
@@ -11,10 +11,16 @@ describe("<TrackCard />", () => {
       artworkUrl100:
         "https://is5-ssl.mzstatic.com/image/thumb/Music116/v4/5a/47/6d/5a476ddd-4690-1297-1896-6a286a497a21/191404113974.png/100x100bb.jpg",
     },
+    isShowDetailsButton: false,
   };
 
   it("should render and match the snapshot", () => {
     const { baseElement } = render(<TrackCard {...trackCardProps} />);
     expect(baseElement).toMatchSnapshot();
   });
+
+  it("should render 1 TrackCard component", () => {
+    const { getByTestId } = render(<TrackCard {...trackCardProps} />);
+    expect(getByTestId("track-card")).toBeInTheDocument();
+  });
```

Comment:
This doesn't really check if there was a trackCard component, this will merely see if there is at the very least something with `track-card` as test-id rendered.

```suggestion
  it("should render 1 TrackCard component", () => {
    const { getAllByTestId } = render(<TrackCard {...trackCardProps} />);
    expect(getAllByTestId("track-card").length).toEqual(1);
  });
```

Above would check if there was at most and at least 1 component rendered.

Reply:
- We won't require this test inside `TrackCard` component, it will be required in `TrackList` container.
- Removing this test from here